common/djangoapps/student/tests/test_verification_status.py

```diff
@@ -35,6 +35,11 @@ class TestCourseVerificationStatus(UrlResetMixin, ModuleStoreTestCase):
 
     PAST = datetime.now(UTC) - timedelta(days=5)
     FUTURE = datetime.now(UTC) + timedelta(days=5)
+    DATES = {
+        'past': PAST,
```

Comment:
nit: this pattern is slightly different from the others (e.g. in common/djangoapps/course_modes/tests/test_models.py and lms/djangoapps/courseware/tests/test_access.py) in which the class variable value is a string and the timestamp value is calculated in the dict initialization. Might be less confusing if they are all consistent.

Reply:
Yeah, there were already a bunch of uses of the PAST and FUTURE constants as datetimes in non-ddt test methods here.  I was trying to keep the scope of changes to a minimum, but it probably does make sense to go ahead and refactor this; I'll change existing usages from `self.PAST` to `self.DATES[PAST]`, etc.